src/server/stream_family.cc

```diff
@@ -144,6 +144,11 @@ inline string NoGroupError(string_view key, string_view cgroup) {
   return absl::StrCat("-NOGROUP No such consumer group '", cgroup, "' for key name '", key, "'");
 }
 
+inline string NoGroupOrKey(string_view key, string_view cgroup, string_view suffix = "") {
```

Comment:
`inline` doesn't do anything here. You can remove it :)

Reply:
removed (removed two other places too)